docs/sql/steampipe-sql.md

```diff
@@ -139,6 +139,24 @@ group by
   runtime;
 ```
 
+You can **remove duplicate rows**:
```

Comment:
maybe 'exclude' is better than 'remove'?

Reply:
yes, agree, that's better. done, thx.